edit-test-case.mdx

```diff
@@ -101,11 +101,31 @@ Click on the `refresh` icon next to the existing **test dependency** and select
   />
 </Frame>
 
-## Run and save edits
+## Save and run edits
 
-`Run` you test case to see if it works. If it's successful, you can click on the `save` button. We will warn you if you try to leave without saving the test.
+`Save & run` you test case to see if it works. You can click on the `run only` button if you want to validate the test first before saving it.
+
+Btw, we will warn you if you try to leave without saving the test case.
 
 <Tip>
-  Double check that your newly edited step is `on` in the test case view if you
-  want to run it in your next **test report**.
+  Double check that your newly edited test is active - the toggle is `on`, if
+  you want to run it in your next **test report**.
```

Comment:
maybe `- the toggle is on. Those active test cases will be executed in your next test report` ?